src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/FeatureFlag/FeatureFlagsFormDataProvider.php

```diff
@@ -110,6 +122,8 @@ public function setData(array $flagsData)
         }
 
         $this->doctrineEntityManager->flush();
+        // force cache regeneration
```

Comment:
```suggestion
        // Clear cache of legacy routes since they can depend on an associated feature flag when the attribute _legacy_feature_flag is used
```